src/include/ipc/dai-intel.h

```diff
@@ -146,7 +146,7 @@ struct sof_ipc_dai_dmic_pdm_ctrl {
  * version number used in configuration data is checked vs. version used by
  * device driver src/drivers/dmic.c need to match. It is incremented from
  * initial value 1 if updates done for the to driver would alter the operation
- * of the microhone.
```

Comment:
This I found hilarious :) you left a typo in the word typo itself in the commit message

Reply:
Haha, I only now noticed it!! Can I still fix this without restarting the approval process?

Reply:
@lgirdwood @kv2019i Should I fix this?

Reply:
@singalsu no need I think. its just the commit message.

Reply:
Good, I'll leave this funny bit into the history.